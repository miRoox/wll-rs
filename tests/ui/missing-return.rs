#[wll::setup]
fn setup() {}

#[wll::teardown]
fn teardown() {}

#[wll::export(do_nothing)]
fn nothing() {}

fn main() {}
