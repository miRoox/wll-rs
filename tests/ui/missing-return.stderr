error: invalid function signature!
 --> $DIR/missing-return.rs:8:1
  |
8 | fn nothing() {}
  | ^^^^^^^^^^^^
